corehq/apps/saved_reports/models.py

```diff
@@ -792,3 +829,98 @@ def get_latest(cls):
             return cls.objects.order_by('-end_datetime')[0]
         except IndexError:
             return None
+
+class SQLReportConfig(SyncSQLToCouchMixin, models.Model):
+    domain = models.CharField(max_length=32, null=False, db_index=True)
```

Comment:
I think this max length has to be longer, at least 128 chars

Reply:
Also wondering about the length of some of the other fields

Reply:
This was the default code generated by `evaluate_couch_model_for_sql`. I still have to fine-tune these variables.

Reply:
Domain should probably be treated as a special case in `evaluate_couch_model_for_sql`, I had thought it already was but see it isn't. We limit domain names in the UI to 25 chars, so it's only domains that are 5+ years old that would have longer names, but they do exist.